tests/lit-tests/vector_type_init-1.ispc

```diff
@@ -0,0 +1,51 @@
+// This test checks that uniform vectors conversion during initialization is perfomed using vector instructions.
+// RUN: %{ispc} %s --target=host -O0 --emit-llvm-text -o - | FileCheck %s
+
+// Vector initialization
+// CHECK: define <4 x double> @test1
```

Comment:
Please add `CHECK-LABEL`s with function name, like this one:
`// CHECK-LABEL: define {{.*}} @bar1`